sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -746,7 +745,7 @@ def wait_for_init(self, *_, node_list=None, verbose=False, timeout=None, **__):
         self.wait_for_nodes_up_and_normal(nodes=node_list)
 
     def upgrade_scylla_cluster(self, new_version: str) -> None:
-        self.k8s_cluster.docker_pull(f"{self.params.get('docker_image')}:{new_version}")
+        self.k8s_clusters[0].docker_pull(f"{self.params.get('docker_image')}:{new_version}")
```

Comment:
maybe leave a comment, if we'll need multiDC upgrade ?, and we should pull on all k8s_clusters ?

Reply:
This is `mini_k8s`. We do not run upgrades on it. I just fixed the interface usage here.